components/backoffice/AttendeeDetail.vue

```diff
@@ -0,0 +1,223 @@
+          :value="registration?.eventName"
+          disabled
+        />
+      </div>
+      <div class="col-start-1">
+        <p class="b1">First name</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.firstName"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Last name</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.lastName"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Username</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.username"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Email</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.email"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Phone</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.phone"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Gender</p>
+        <input
+          type="text"
+          class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 text-[#818181] shadow-inner"
+          :value="registration?.gender"
+          disabled
+        />
+      </div>
+      <div class="">
+        <p class="b1">Checked in</p>
+        <!-- <input type="text" class="b2 border-1 my-4 w-full rounded-lg bg-[#F8FBFF] px-4 py-2 shadow-inner"
+        :value="registration?.status" /> -->
+        <div class="" v-show="isShowStatus">
+          <div class="relative">
+            <button
+              @click="isShowStatusBy = !isShowStatusBy"
+              class="b2 border-1 my-4 flex w-full items-center justify-between gap-1 rounded-lg bg-[#F8FBFF] px-4 py-2 shadow-inner"
+            >
+              {{
+                selectedStatus.status
+                  ? selectedStatus.status
+                  : registration?.status
+              }}
+              <Arrow
+                :class="`${isShowStatusBy ? 'rotate-90' : '-rotate-90'} text-sm duration-200`"
+              />
+            </button>
+            <div
+              v-if="isShowStatusBy"
+              class="absolute top-12 z-20 flex w-full flex-col items-start gap-1 rounded-xl bg-[#fdfdfd] p-3 text-start drop-shadow-lg"
+            >
+              <button
```

Comment:
this pls add another cursor design if this can't click kub

Reply:
where can't click wa?

Reply:
![image](https://github.com/user-attachments/assets/1249bc6c-19c7-464c-8bd7-42fd9a9add8d)
Going it can click mai na?